README.md

```diff
@@ -78,6 +78,17 @@ Username for the first super user.
 #### FIRST_SUPERUSER_PASSWORD
 Password for the first super user.
 
+### Pruner Configuration
+
+---
+
+#### ENABLE_PRUNER
```

Comment:
I think you should show an empty toml file to help explain what it should look like.

Reply:
yes i've added pruner_config.toml to the repo

Reply:
Sorry, I mean show the configuration file in the readme.

Reply:
yes will add the link in the readme